be/src/runtime/local_tablets_channel.cpp

```diff
@@ -265,6 +268,10 @@ void LocalTabletsChannel::add_chunk(Chunk* chunk, const PTabletWriterAddChunkReq
     // This will only block the bthread, will not block the pthread
     count_down_latch.wait();
 
+    // Abort tablets which primary replica already failed
+    _abort_replica_tablets(request, response->status().status_code() == 0 ? "" : response->status().error_msgs()[0],
```

Comment:
Why to call `_abort_replica_tablets` when `response->status().status_code() == 0`?